lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -173,8 +174,10 @@ class MailboxController extends BaseMailboxController {
             _createNewMailboxSuccess(success);
           } else if (success is SearchMailboxSuccess) {
             _searchMailboxSuccess(success);
-          } else if (success is DeleteMultipleMailboxSuccess) {
-            _deleteMailboxSuccess(success);
+          } else if (success is DeleteMultipleMailboxAllSuccess) {
+            _deleteMultipleMailboxSuccess(success.listMailboxIdDeleted, success.currentMailboxState);
+          } else if (success is DeleteMultipleMailboxHasSomeSuccess) {
+            _deleteMultipleMailboxSuccess(success.listMailboxIdDeleted, success.currentMailboxState);
```

Comment:
why not?
```suggestion
          } else if (success is DeleteMultipleMailboxAllSuccess || success is DeleteMultipleMailboxHasSomeSuccess) {
            _deleteMultipleMailboxSuccess(success.listMailboxIdDeleted, success.currentMailboxState);
          }
```

Reply:
It is not the same variable. So we can't be like you.